web/subscribers/index.php

```diff
@@ -4,32 +4,24 @@
 
 // Default parameters if not specified
 $defaults = [
-    "color" => "blue",
+    "color" => "brightgreen",
     "logo" => "youtube",
     "logoColor" => "white",
-    "style" => "flat-square",
+    "style" => "plastic",
```

Comment:
The default styles and colors shouldn't be changed unless you have a good reason to change them. (If we do keep these changes, the README will have to be updated.) I think the defaults were fine before.

Reply:
Hey, @DenverCoder1, I have replaced these parameters with the default ones, as required...